docker/wsgi.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from mms import mxnet_model_server
+import sys
+import json
+import os
+from mms.arg_parser import ArgParser
+
+mms_arg_header = 'MMS Argument'
+args = []
+found_mms_args = 0
+
+mms_config_path = os.environ['MXNET_MODEL_SERVER_CONFIG']
+is_gpu_image = os.environ['GPU_IMAGE']
+
+
+def read_models_from_file():
+    """
+    This method reads the model's meta-data from a local file. This is used for MMS in
```

Comment:
more than one model? `models'` or just `reads model metadata`...

Reply:
it reads all the models metadata. Wrong usage of '